app/controller/app_controller.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+require 'singleton'
+require_relative '../service/bank_service'
+require_relative '../service/atm_service'
+require_relative '../models/user'
+
+class AppController
+  include Singleton
+
+  def initialize
+    @@bank_service = BankService.instance
+    @@atm_service = ATMService.instance
+  end
+
+  def start
+    while true
+      puts "Welcome to the bank"
+      puts "Please select an option"
+      puts "1. Create an account"
+      puts "2. Select an ATM"
+      puts "3. Exit"
+      operation = gets.chomp
+      case operation
+      when "1"
+        create_account
+      when "2"
+        select_atm
+      when "3"
+        exit 0
+      else
+        puts "Invalid option"
+      end
+      gets.chomp
+      clear_console
+    end
+  end
+
+  def create_account
+    clear_console
+    puts "Create an account"
+    print "Enter your email: "
+    email = gets.chomp
+    print "Enter your name: "
+    name = gets.chomp
+    print "Enter your job: "
+    job = gets.chomp
+    print "Enter your address: "
+    address = gets.chomp
+    print "Enter your pin: "
+    pin = gets.chomp
+    response = @@bank_service.create_account(User.new(email, name, job, address), pin)
```

Comment:
You could move this in a separate method like `collect_user_input` 

```
data = {}

%w[email name job address pin].each do |input| 
  print "Enter your #{input}: "
  user_input = gets.chomp

  data[input] = user_input 
end
```

and use it something like this: `create_account(User.new(*data.except(:pin).values), data[:pin])`